spec/controllers/locations_controller_spec.rb

```diff
@@ -0,0 +1,19 @@
+RSpec.describe LocationsController, type: :controller do
+  describe 'POST #index' do
+    context 'when performing an installation search' do
+      it 'assigns @search' do
+        post :index, params: { query: 'installation' }
+
+        expect(assigns(:search).query).to eq('installation')
+      end
+    end
+
+    context 'when performing a ZIP code search' do
+      it 'assigns @search' do
+        post :index, params: { query: '20010' }
+
+        expect(assigns(:search).query).to eq('20010')
+      end
+    end
+  end
+end
```

Comment:
Backfilling a couple un-covered pieces of code with a controller spec.

I initially did a unit-style test against each of the models, but changed that to this controller spec. RSpec documentation discourages the use of controller tests (in favor of request specs), but the `query` method on these two search classes isn't used (currently) elsewhere in the code. I chose a controller spec since it was the highest level spec that could reasonably cover the method.